tests/test_keyval.py

```diff
@@ -1,11 +1,75 @@
+import os
 import pytest
-
+import sys
+import time
+from multiprocessing import Process
 from pathlib import Path
 
+from liquidctl.keyval import RuntimeStorage, _FilesystemBackend
 
-def test_fs_backend_handles_values_corupted_with_nulls(tmpdir, caplog):
-    from liquidctl.keyval import _FilesystemBackend
 
+@pytest.fixture
+def tmpstore(tmpdir):
+    run_dir = tmpdir.mkdir('run_dir')
+    prefixes = ['prefix']
+
+    backend = _FilesystemBackend(key_prefixes=prefixes, runtime_dirs=[run_dir])
+    return RuntimeStorage(prefixes, backend=backend)
+
+
+def test_loads_and_stores(tmpstore):
+    assert tmpstore.load('key') is None
+    assert tmpstore.load('key', default=42) == 42
+
+    tmpstore.store('key', '42')
+
+    assert tmpstore.load('key') == '42'
+    assert tmpstore.load('key', of_type=int) is None
+
+
+def test_updates_with_load_store(tmpstore):
+    assert tmpstore.load_store('key', lambda x: x) == (None, None)
+    assert tmpstore.load_store('key', lambda x: x, default=42) == (None, 42)
+    assert tmpstore.load_store('key', lambda x: str(x)) == (42, '42')
+    assert tmpstore.load_store('key', lambda x: x, of_type=int) == ('42', None)
```

Comment:
These are a good test of tests, I did not think of those extra cases.

Reply:
Yes, I wanted to have something that pinned down the behavior around `default` and `of_type`, particularly now that this also interacts with the `load_store` closure.

If we were to document `load_store`, it would say something like: returns a tuple of the old parsed value (before applying default due to None or of_type failure) and the newly stored value (the result of applying `of_type` and `default` and passing that through the supplied closure).